x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/GetDeploymentStatsAction.java

```diff
@@ -115,331 +106,6 @@ public int hashCode() {
 
         public static final ParseField DEPLOYMENT_STATS = new ParseField("deployment_stats");
 
-        public static class AllocationStats implements ToXContentObject, Writeable {
```

Comment:
This is is used by two transport actions now, I opted to move it to its own class. It is pretty big regardless and probably needed its own class.